src/components/report/ReportNonSighting.js

```diff
@@ -0,0 +1,93 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { Formik, Form } from 'formik';
+import moment from 'moment';
+import * as yup from 'yup';
+
+import Loader from '../helpers/Loader';
+import Error from '../helpers/Error';
+import NonSightingDetailsFieldset from './fieldset/NonSightingDetailsFieldset';
+import ExpectationsFieldset from './fieldset/ExpectationsFieldset';
+import ContributorFieldset from './fieldset/ContributorFieldset';
+import FurtherInformationFieldset from './fieldset/FurtherInformationFieldset';
+import SubmitFieldset from './fieldset/SubmitFieldset';
+import { getReportNonSightingOptions, postReportNonSighting } from '../../actions/reportNonSighting';
+
+const initialValues = {
+  dateTimeSighted: moment(),
+  location_details: '',
+  expectations: '',
+  contributor: {
+    name: '',
+    email: '',
+    phone: '',
+    activity: '',
+    heard: '',
+    communications: false,
+  },
+  comments: '',
+};
```

Comment:
Initial value for every field should be passed to Formik or otherwise React complains when I input some value that I'm changing uncontrolled input to controlled.